### PR TITLE
Add integration tests for user response during registration

### PR DESCRIPTION
**Pull request by danmyway**
Integration test for #390.

During registration, while no username nor password was provided check if user prompt requests input for said data.
Then continue with the registration properly.

### CODE REVIEW

**Comment by r0x0d** [comment]
@danmyway if you find it useful, I think the same logic you applied to the integration test for username/password, you can apply to the `org` parameter as well https://github.com/oamg/convert2rhel/pull/390/files#diff-51e9ff11d39778d3b26778b293fc350430a82e2feed0dab2938da7c0069ffdc6R146-R147 since I have done the same `while` loop to that parameter too.

File: tests/integration/tier0/check-user-response/test_user_response.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    # check for user prompt enforcing input, then continue with registration
+    with convert2rhel(
+        "-y --no-rpm-va --serverurl {}".format(
+            env.str("RHSM_SERVER_URL"),
+        )
+    ) as c2r:
+        c2r.expect(" ... activation key not found, username and password required")
+        c2r.expect("Username: ")
+        c2r.sendline()
+        assert c2r.expect("Username: ") == 0
```
Comment (mkluson) [inline tests/integration/tier0/check-user-response/test_user_response.py:16]: what is the difference between `c2r.expect("Username: ")` and `assert c2r.expect("Username: ") == 0`

**Comment by r0x0d** [comment]
> This is a test for #390 it would be great to have it in the same PR, so the test runs on the fixed code. Is it possible to merge it together somehow? (what do you think @kokesak @SpyTec?)

I can rebase this PR into mine, @mkluson. @danmyway should have push access to that branch since it's public.

**Comment by mkluson** [comment]
> I can rebase this PR into mine, @mkluson. @danmyway should have push access to that branch since it's public.

@danmyway can you push your changes to that PR and close this one?

**Comment by danmyway** [comment]
> > I can rebase this PR into mine, @mkluson. @danmyway should have push access to that branch since it's public.
> 
> @danmyway can you push your changes to that PR and close this one?

Sure.

**Pull request closed without merging**